data-plane/core/src/test/java/dev/knative/eventing/kafka/broker/core/tracing/TracingConfigTest.java

```diff
@@ -39,12 +39,14 @@ public void shouldParseConfigGivenDirectory() throws IOException {
     write(dir, "/backend", " zipkin ");
     write(dir, "/sample-rate", " 0.1 ");
     write(dir, "/zipkin-endpoint", "  http://localhost:9241/v2/api/spans/     ");
+    write(dir, "/heads-format", " b3-multi-header  ");
```

Comment:
We should test also w3 (default) and b3 single header, we can keep this test as it was and assert that `HeadersFormat` is `W3C` and add these 2 more tests:
```java
  @Test
  public void shouldParseConfigB3SingleHeaderGivenDirectory() throws IOException {

    final var dir = Files.createTempDirectory("tracing");

    write(dir, "/backend", " zipkin ");
    write(dir, "/sample-rate", " 0.1 ");
    write(dir, "/zipkin-endpoint", "  http://localhost:9241/v2/api/spans/     ");
    write(dir, "/headers-format", " b3-single-header  ");

    final var config = TracingConfig.fromDir(dir.toAbsolutePath().toString());

    assertThat(config.getBackend()).isEqualTo(Backend.ZIPKIN);
    assertThat(config.getSamplingRate()).isEqualTo(0.1F);
    assertThat(config.getUrl()).isEqualTo("http://localhost:9241/v2/api/spans/");
    assertThat(config.getHeadsFormat()).isEqualTo(HeadersFormat.B3_SINGLE_HEADER);
  }

  @Test
  public void shouldParseConfigB3MultiHeaderGivenDirectory() throws IOException {

    final var dir = Files.createTempDirectory("tracing");

    write(dir, "/backend", " zipkin ");
    write(dir, "/sample-rate", " 0.1 ");
    write(dir, "/zipkin-endpoint", "  http://localhost:9241/v2/api/spans/     ");
    write(dir, "/headers-format", " b3-multi-header  ");

    final var config = TracingConfig.fromDir(dir.toAbsolutePath().toString());

    assertThat(config.getBackend()).isEqualTo(Backend.ZIPKIN);
    assertThat(config.getSamplingRate()).isEqualTo(0.1F);
    assertThat(config.getUrl()).isEqualTo("http://localhost:9241/v2/api/spans/");
    assertThat(config.getHeadsFormat()).isEqualTo(HeadersFormat.B3_MULTI_HEADER);
  }
```

Reply:
reasonable, I've updated the PR.